tools/check_format.py

```diff
@@ -29,48 +29,44 @@
 # For now it includes the validation server but that really should be injected too.
 REAL_TIME_WHITELIST = ('./source/common/common/utility.h',
                        './source/common/event/real_time_system.cc',
-                       './source/common/event/real_time_system.h',
-                       './source/exe/main_common.cc',
-                       './source/exe/main_common.h',
-                       './source/server/config_validation/server.cc',
+                       './source/common/event/real_time_system.h', './source/exe/main_common.cc',
+                       './source/exe/main_common.h', './source/server/config_validation/server.cc',
                        './source/common/common/perf_annotation.h',
                        './test/test_common/simulated_time_system.cc',
                        './test/test_common/simulated_time_system.h',
-                       './test/test_common/test_time.cc',
-                       './test/test_common/test_time.h',
-                       './test/test_common/utility.cc',
-                       './test/test_common/utility.h',
+                       './test/test_common/test_time.cc', './test/test_common/test_time.h',
+                       './test/test_common/utility.cc', './test/test_common/utility.h',
                        './test/integration/integration.h')
 
 CLANG_FORMAT_PATH = os.getenv("CLANG_FORMAT", "clang-format-7")
 BUILDIFIER_PATH = os.getenv("BUILDIFIER_BIN", "$GOPATH/bin/buildifier")
 ENVOY_BUILD_FIXER_PATH = os.path.join(
     os.path.dirname(os.path.abspath(sys.argv[0])), "envoy_build_fixer.py")
-HEADER_ORDER_PATH = os.path.join(
-    os.path.dirname(os.path.abspath(sys.argv[0])), "header_order.py")
+HEADER_ORDER_PATH = os.path.join(os.path.dirname(os.path.abspath(sys.argv[0])), "header_order.py")
 SUBDIR_SET = set(common.includeDirOrder())
 INCLUDE_ANGLE = "#include <"
 INCLUDE_ANGLE_LEN = len(INCLUDE_ANGLE)
 
 PROTOBUF_TYPE_ERRORS = {
     # Well-known types should be referenced from the ProtobufWkt namespace.
-    "Protobuf::Any":                    "ProtobufWkt::Any",
-    "Protobuf::Empty":                  "ProtobufWkt::Empty",
-    "Protobuf::ListValue":              "ProtobufWkt::ListValue",
-    "Protobuf::NULL_VALUE":             "ProtobufWkt::NULL_VALUE",
-    "Protobuf::StringValue":            "ProtobufWkt::StringValue",
-    "Protobuf::Struct":                 "ProtobufWkt::Struct",
-    "Protobuf::Value":                  "ProtobufWkt::Value",
+    "Protobuf::Any": "ProtobufWkt::Any",
```

Comment:
I'm OK with losing the table-ness of this if needed, but just curious if there is a way to disable the formatter for a region, such as we do in https://github.com/envoyproxy/envoy/blob/879ccac00da73d4b3343291d4c8c0839ff40d920/source/common/http/codes.cc#L143

Reply:
Yes -- IMO it's more readable to leave it formatted as a table but I think the google style guide might disagree.